upgrade.yml

```diff
@@ -418,6 +418,14 @@
   any_errors_fatal: true
   tags: swift
 
+  pre_tasks:
+    # FIXME Added since we are changing swift init scripts in 3.0.x
+    # This will stop services started by "old" scripts. This can be removed
+    # after all systems have migrated to 3.0.x
+    - name: stop swift services started by swift-init
+      command: swift-init stop all
+      failed_when: false
```

Comment:
In PR that I had related to swift upstart changes, @j2sol had recommended using failed_when: false instead of ignore_errors: true